src/Actions/NotifyUsersWorkflowAction.php

```diff
@@ -150,18 +164,28 @@ public function execute(WorkflowInstance $workflow)
                 $email = Email::create();
                 try {
                     $email->setTo($member->Email);
-                } catch (Swift_RfcComplianceException $exception) {
+                } catch (RfcComplianceException) {
                     // If the email address isn't valid we should skip it rather than break
                     // the rest of the processing
                     continue;
                 }
                 $email->setSubject($subject);
-                $email->setFrom($from);
+                if ($from) {
+                    $email->setFrom($from);
+                }
                 $email->setBody($body);
                 $email->send();
+                $emailSent = true;
             }
         }
 
+        if (!$emailSent) {
+            Injector::inst()->get(LoggerInterface::class)->warning(
+                'No email was sent for notify users workflow action '
+                . "'{$this->Title}' in workflow '{$this->WorkflowDef()?->Title}'."
+            );
+        }
```

Comment:
This warning doesn't resolve any of the listed issues, but adds logging consistent with other logging I'm adding in this PR so that the notify users action can either be updated or removed as appropriate.